src/Yesod/Markdown.hs

```diff
@@ -50,7 +50,6 @@ import Text.Blaze.Html (preEscapedToMarkup)
 import Text.HTML.SanitizeXSS (sanitizeBalance)
 import Text.Hamlet (hamlet, Html)
 import Text.Pandoc
-import Text.Pandoc.Error
```

Comment:
Yeah, it appears this would require us to bump the lower bounds as well. I'll remove this commit, since it's not really necessary.